rvm.el

```diff
@@ -344,6 +344,15 @@ function."
 (defun rvm--emacs-gempath ()
   (getenv "GEM_PATH"))
 
+(defun rvm--emacs-bundlepath ()
+  (let ((bundle-config (rvm--call-bundle-process "config get path"))
+        (search-start 0)
+        (bundle-paths '()))
+    (while (string-match ": \"?\\([^\"]*\\)\"?$" bundle-config search-start)
```

Comment:
Supporting older versions means looking for `"^path$"` then moving down 1 line attempting to match this regex. 

Would this be more accurate: `": \"\\([^\"]+\\)\"$"`? Don't we always want quotes with 1 or more chars between them or is there a reason to make it more lax?

Reply:
No, there wasn’t any reason. I make it lax at the beginning of my tests as I was not very sure about the bundle config output format. However, now that I more confident about it, you’re right we can make it more strict.

Reply:
> When I test old bundler version on my machine, the output format is always the same. Can you copy/paste an exemple of a different output you encounter?

Yes you're right. I forgot that one can do `bundle config path` with older versions. My comment assumed we'd have to parse the output of `bundle config`

Reply:
Oh, ok I understand. Thus no problem there :)